.gitignore

```diff
@@ -139,6 +139,15 @@ tools/*/*.i.tmp
 # === Rules for Windows vcbuild.bat ===
 /temp-vcbuild
 
+# === Rules for CMake ===
+cmake-build-debug/
+CMakeCache.txt
+CMakeFiles
+CTestTestfile.cmake
+cmake_install.cmake
+install_manifest.txt
+*.cbp
```

Comment:
do you have a reference on this extension?

Reply:
More of clion specific, I have no idea what's about. But pretty sure it's useless.

![image](https://user-images.githubusercontent.com/3759816/111629534-5c18cd80-882c-11eb-9f9d-faf115a704b8.png)

Reply:
Also most content from our previous experience with uvwasi in https://github.com/nodejs/uvwasi/blob/8832fdcbc02a19972aa976962b677b15872bbe9d/.gitignore#L57.